### ISSUE TITLE
🧑‍💻: Development of News section

### ISSUE THREAD

**Issue by Avdhesh-Varshney** [Avdhesh-Varshney/chanakya-niti#147]
:red_circle: **Title** : News related to Acharya Chanakya 
:red_circle: **Enhancement Aim** : Fetch the latest news related to chanakya and provide a detailed news link and small desciption and title of the news. Categories the news if possible i.e. related to teachings, economics, politics, history, family, security, etc.
:red_circle: **Changes** : Some news links for references,
- [News 1](https://www.msn.com/en-in/news/India/three-crucial-advices-from-chanakyaniti-for-a-successful-life/ar-BB1nvKCC?ocid=msedgntp&pc=NMTS&cvid=85dea7aee7e048928cc879731d166c31&ei=15)
- [News 2](https://www.msn.com/en-in/news/world/what-chanakya-says-about-head-of-a-family-and-qualities-they-must-have/ar-BB1nZDTf?ocid=msedgntp&pc=NMTS&cvid=4b865bf82bdd4814813cf2ae7a31bdb8&ei=38)

### Screenshots 📷

N/A

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- What is your participant role? (Mention the Open Source Program name. Eg. GSOC, GSSOC, SSOC, JWOC, etc.)

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by dfghjkbd** [comment]
please assign me this issue

**Comment by Avdhesh-Varshney** [comment]
@dfghjkbd mention the required details. 

- Full name :
- What is your participant role?

**Comment by MetiUmashri** [comment]
Can you please assign this to me under GSSOC

**Comment by Asymtode712** [comment]
@Avdhesh-Varshney As @dfghjkbd haven't responded since 5 days, can I work on this problem??

**Comment by Avdhesh-Varshney** [comment]
@Asymtode712 go ahead.

**Comment by Asymtode712** [comment]
https://github.com/Avdhesh-Varshney/chanakya-niti/assets/115717746/70c55e50-66f7-4812-98b7-2abedca9824c

@Avdhesh-Varshney How bout this is it looking good?

**Comment by Avdhesh-Varshney** [comment]
@Asymtode712 Looks great

**Comment by Asymtode712** [comment]
@Avdhesh-Varshney So currently I am using an API for avoiding CORS once it is merged into frontend, I would modify the backend so that the API wouldn't be required!.....

So should I raise a PR??
